pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -613,11 +632,20 @@ func (c *Controller) getProxyServiceInstancesByPod(pod *v1.Pod, service *v1.Serv
 			log.Warnf("Failed to find port for service %s/%s: %v", service.Namespace, service.Name, err)
 			continue
 		}
-		// There is only one IP for kube registry
+
 		proxyIP := proxy.IPAddresses[0]
```

Comment:
If we now have multiple IPs then why are we getting `[0]`?

Reply:
Here I'm taking the first IP and using it to generate the original ServiceInstance，then a deep copy on multiple IP can make based on this.

Reply:
Already modified